libshaderc/src/shaderc_private.h

```diff
@@ -23,8 +23,10 @@
 
 // Described in shaderc.h.
 struct shaderc_spv_module {
-  // SPIR-V binary.
-  std::string spirv;
+  // Compilation output data.
```

Comment:
Should say more about what kinds of things this could contain. 
It can be a vector of SPIR-V binary module words, or a string (null terminated or not?).
And when it is a string, do we guarantee what the bytes are after the "size" bytes but within the last elements of the vector?